nilearn/signal.py

```diff
@@ -168,22 +166,22 @@ def _detrend(signals, inplace=False, type="linear", n_batches=10):
 def _check_wn(btype, freq, nyq):
     wn = freq / float(nyq)
     if wn >= 1.:
-        warnings.warn(
-            'The frequency specified for the %s pass filter is '
-            'too high to be handled by a digital filter (superior to '
-            'nyquist frequency). It has been lowered to %.2f (nyquist '
-            'frequency).' % (btype, nyq))
         # results looked unstable when the critical frequencies are
         # exactly at the Nyquist frequency. See issue at SciPy
         # https://github.com/scipy/scipy/issues/6265. Before, SciPy 1.0.0 ("wn
         # should be btw 0 and 1"). But, after ("0 < wn < 1"). Due to unstable
         # results as pointed in the issue above. Hence, we forced the
         # critical frequencies to be slightly less than 1. but not 1.
         wn = 1 - 10 * np.finfo(1.).eps
+        warnings.warn(
+            'The frequency specified for the %s pass filter is '
+            'too high to be handled by a digital filter (superior to '
+            'nyquist frequency). It has been lowered to %.2f (nyquist '
+            'frequency).' % (btype, wn))
     return wn
 
 
-def butterworth(signals, sampling_rate, low_pass=None, high_pass=None,
+def butterworth(signals_in, sampling_rate, low_pass=None, high_pass=None,
```

Comment:
Why did you change this? We'd rather keep the name "signals".

Reply:
I originally did it to disambiguate it from the signal import from scipy, but now that I renamed that also, this is not necessary :)

Reply:
This is addressed in PR #1656